src/test/crush/crush-classes.sh

```diff
@@ -71,7 +71,12 @@ function TEST_classes() {
     #
     ceph osd getcrushmap > $dir/map || return 1
     crushtool -d $dir/map -o $dir/map.txt || return 1
-    sed -i \
+    if [ `uname` = FreeBSD ]; then
```

Comment:
how about `src/test/mon/osd-crush.sh` , `src/test/test_pool_create.sh` and `src/test/test_crush_bucket.sh`, maybe we can define a function in `qa/workunits/ceph-helpers.sh` to normalize the difference between FreeBSD sed and GNU sed?